h2o-core/src/main/java/water/JettyHTTPD.java

```diff
@@ -197,6 +197,11 @@ else if (H2O.ARGS.kerberos_login) {
         System.setProperty("java.security.auth.login.config",H2O.ARGS.login_conf);
         loginService = new JAASLoginService("krb5loginmodule");
       }
+      else if (H2O.ARGS.pam_login) {
+        Log.info("Configuring JAASLoginService (with PAM)");
+        System.setProperty("java.security.auth.login.config",H2O.ARGS.login_conf);
```

Comment:
Yes, there are two possible ways and only the one using system property is applicable, see: http://docs.oracle.com/javase/7/docs/technotes/guides/security/jgss/tutorials/LoginConfigFile.html